### DISCUSSION TITLE
gogoanime being slow?

### DISCUSSION THREAD

**Discussion by Quicksilver151** [sdaqo/anipy-cli#145] [General]
is it just me or is gogoanime super slow now. it refuses to properly connect even. anipy always time out

**Comment by sdaqo** [comment]
Probably because the gogo ajax server is down (this is needed for loading the episode list). Their site also does not work... Proably just gotta wait until they fix their server
![Screenshot_20231129_010929](https://github.com/sdaqo/anipy-cli/assets/63876564/e9705e07-d81e-4aff-ba63-1e9cad4449cb)

**Reply by Quicksilver151** [reply]
alr. looks like it works now. great
